src/showcase/stories/angular/sidebar/sidebar-close-button/sidebar-close-button.stories.ts

```diff
@@ -1,9 +1,48 @@
 import { SbbSidebarCloseButton } from '@sbb-esta/lyne-angular/sidebar/sidebar-close-button';
 import { Args, argsToTemplate, Meta } from '@storybook/angular';
+import { ArgTypes, InputType, StoryContext } from '@storybook/types';
+
+const size: InputType = {
+  control: {
+    type: 'inline-radio',
+  },
+  options: ['l', 'm', 's'],
+};
+
+const type: InputType = {
+  control: {
+    type: 'select',
+  },
+  options: ['button', 'reset', 'submit'],
+};
+
+const value: InputType = {
+  control: {
+    type: 'text',
+  },
+};
+
+const argTypes: ArgTypes = {
+  size,
+  type,
+  value,
+};
+
+const args: Args = {
+  size: size.options![0],
+  type: type.options![0],
+};
 
 const meta: Meta = {
   title: 'elements/sbb-sidebar/sbb-sidebar-close-button',
   component: SbbSidebarCloseButton,
+  parameters: {
+    actions: { handles: ['click'] },
+    backgroundColor: (context: StoryContext) =>
+      context.args['negative'] ? 'var(--sbb-color-charcoal)' : 'var(--sbb-color-white)',
```

Comment:
we shouldn't have negative documented as in this context it shoulnd't be used

Reply:
ok I see they were generated anyways, so it's better to have them in a correct state :). I think we should just disabled all the unmatching controls. maybe just compare with lyne-elements storybook?